test/once.js

```diff
@@ -0,0 +1,73 @@
+var tape = require('tape')
+
+module.exports = function (observable) {
+
+  tape('once', function (t) {
+    var o = observable()
+    var fired = 0
+    o.once(function (v) {
+      fired ++
+      t.equal(v, 1)
+    })
+
+    o.set(1)
+
+    o.once(function (v) {
+      fired ++
+      t.equal(v, 2)
+    }, false)
+
+    o.once(function (v) {
+      fired ++
+      t.equal(v, 1)
+    })
+
+    t.equal(fired, 2)
+
+    o.set(2)
+    o.set(3)
+
+    t.equal(fired, 3)
+
+    t.end()
+  })
+
+  tape('once, !immediately, inside trigger', function (t) {
+    var o = observable()
+    var fired = 0
+    o.once(function (v) {
+      t.equal(v, 1)
+      fired ++
+      o.once(function (v) {
+        fired ++
+        t.equal(v, 2)
+      }, false)
+    })
+
+    o.set(1)
+    t.equal(fired, 1)
+    o.set(2)
+    t.equal(fired, 2)
+    t.end()
+  })
+
+  tape('once, remove', function (t) {
```

Comment:
this is the new test.
I moved all the `once` related tests to own file as they're a specific subset of features